Tests/test_pickle.py

```diff
@@ -88,3 +82,14 @@ def test_pickle_la_mode_with_palette(tmp_path):
 
         im.mode = "PA"
         assert im == loaded_im
+
+
+def test_pickle_tell():
```

Comment:
Thanks!

I should have also added this:

```suggestion
@skip_unless_feature("webp")
def test_pickle_tell():
```

Reply:
Yep, I've pushed that to my PR